src/main/java/org/vafer/jdependency/Clazzpath.java

```diff
@@ -77,56 +76,54 @@ public boolean removeClazzpathUnit( final ClazzpathUnit pUnit ) {
      * @return newly created {@link ClazzpathUnit} with id of pFile.absolutePath
      * @throws IOException
      */
-    public final ClazzpathUnit addClazzpathUnit( final File pFile ) throws IOException {
-        return addClazzpathUnit(pFile, pFile.getAbsolutePath());
+    public ClazzpathUnit addClazzpathUnit(final File pFile) throws IOException {
```

Comment:
Is there a reason you removed the `final`?

Reply:
Class itself is `final` already